hack/test-end-to-end-docker.sh

```diff
@@ -6,10 +6,6 @@ source "$(dirname "${BASH_SOURCE}")/lib/init.sh"
 
 os::log::info "Starting containerized end-to-end test"
 
-# cluster up no longer produces a cluster that run the e2e test.  These use cases are already mostly covered
```

Comment:
is oc cluster up in a state where this can work?
/cc @deads2k

Reply:
> is oc cluster up in a state where this can work?
> /cc @deads2k

He's manually re-installing (without an operator), the registry down below.  It may work, depending on whether the command runs or not.  I think the larger question is whether this script should exist in origin when it is focused on testing the registry, not origin.

Reply:
it's living here until we have time to move it properly, it's tech debt, I agree, but not tech debt we can afford to resolve right now.

Also it appears to be currently finding bugs in deployments.

Reply:
> it's living here until we have time to move it properly, it's tech debt, I agree, but not tech debt we can afford to resolve right now.
> 
> Also it appears to be currently finding bugs in deployments.

The email chain about a restmapping cache needing to catch up?  That's not unique.  You're seeing it here first only because the e2e jobs aren't checking the binaries we're shipping.